apps/dashboard/src/main.tsx

```diff
@@ -106,6 +106,30 @@ const router = createBrowserRouter([
         path: ROUTES.ROOT,
         element: <DashboardRoute />,
         children: [
+          {
+            path: 'topics',
+            element: <RedirectToEnvironment targetRoute={ROUTES.TOPICS} />,
+          },
+          {
+            path: 'workflows',
+            element: <RedirectToEnvironment targetRoute={ROUTES.WORKFLOWS} />,
+          },
+          {
+            path: 'subscribers',
+            element: <RedirectToEnvironment targetRoute={ROUTES.SUBSCRIBERS} />,
+          },
+          {
+            path: 'api-keys',
+            element: <RedirectToEnvironment targetRoute={ROUTES.API_KEYS} />,
+          },
+          {
+            path: 'environments',
+            element: <RedirectToEnvironment targetRoute={ROUTES.ENVIRONMENTS} />,
+          },
+          {
+            path: 'activity-feed',
+            element: <RedirectToEnvironment targetRoute={ROUTES.ACTIVITY_FEED} />,
+          },
```

Comment:
Right now we pretty much hard code those items, can we somehow do it programticaly, in case more endpoints to the /env/:envId namespace will be added later? Please note to only do this for top level, avoiding things like /env/:envId/workflow/:workflowId, etc...